PyProgress is broken in demo

<!-- For bugs or other problems please provide the following details in addition to
     your issue report, if applicable. See also https://wxpython.org/pages/how-to-submit-issue/

     For issues about building on Linux, please read this page before reporting it here:
     https://wxpython.org/blog/2017-08-17-builds-for-linux-with-pip/
-->

**Operating system**: Windows 10, 64-bit
**wxPython version & source**: wxpython extras wheel, 4.1.1        <!-- pypi, self-built, etc. -->
**Python version & source**:  python 3.9, from python.org          <!-- stock, anaconda, EDM, distro, self-built, etc. -->

**Description of the problem**:

<!-- if possible please include a small runnable application in the details section below that demonstrates the problem -->
In the wxpython demo, the pyprogress is broken with the below error:

<details>
  <summary>Error message (click to expand)</summary>

```python
Traceback (most recent call last):
  File "agw\PyProgress.py", line 107, in OnStartProgress
    dlg = PP.PyProgress(None, -1, "PyProgress Example",
  File "C:\Python39\lib\site-packages\wx\lib\agw\pyprogress.py", line 495, in __init__
    buttonSizer.Add(self._btnAbort, 0, sizerFlags, LAYOUT_MARGIN)
wx._core.wxAssertionError: C++ assertion "!(flags & wxALIGN_RIGHT)" failed at ..\..\src\common\sizer.cpp(2168) in wxBoxSizer::DoInsert(): Horizontal alignment flags are ignored in horizontal sizers

```
</details>

--- a/wx/lib/agw/pyprogress.py
+++ b/wx/lib/agw/pyprogress.py
@@ -481,7 +481,7 @@ class PyProgress(wx.Dialog):
         sizeDlgModified = False
 
         if wx.Platform == "__WXMSW__":
-            sizerFlags = wx.ALIGN_RIGHT|wx.ALL
+            sizerFlags = wx.ALL
         else:
             sizerFlags = wx.ALIGN_CENTER|wx.BOTTOM|wx.TOP
 

